### PR TITLE
Fix compatibility with PHPUnit 8

### PR DESCRIPTION
**Pull request by jderusse**
The method's signature changed in https://github.com/sebastianbergmann/phpunit/commit/f5e5add13e73933c65878f53d5a94cdc0f120cc4

### CODE REVIEW

File: src/BaseBundleTestCase.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ private function ensureKernelShutdown()
     /**
      * Clean up Kernel usage in this test.
      */
-    protected function tearDown()
+    protected function tearDown(): void
```
Comment (xabbuh) [inline src/BaseBundleTestCase.php]: This will break on PHP 5 (and 7.0). Should we use the `@after` annotation instead?

**Comment by hugo-goncalves-kununu** [comment]
Hi guys, will you create a new release with this PR anytime soon? Thanks!

**Comment by stof** [comment]
@Nyholm any news here ?

**Comment by Nyholm** [comment]
Thank you @jderusse for the PR and @xabbuh.

I assume that @hugo-goncalves-kununu and @stof also helped reviewing this. 
I think this looks good.

**Review by Nyholm — APPROVED** [review]

**Comment by stof** [comment]
@Nyholm I haven't reviewed anything. I discoevered this PR 40 minutes ago when wanting to report the incompatibility.

**Comment by hugo-goncalves-kununu** [comment]
Hi @Nyholm! This commit works fine, but can you create a new release please? So that we don't need to use dev-master in composer.json? Thank you

**Pull request merged**